lib/core/src/persist/chain.rs

```diff
@@ -199,6 +199,15 @@ impl Persister {
         )
     }
 
+    pub(crate) fn list_pending_refund_chain_swaps(&self) -> Result<Vec<ChainSwap>> {
+        let con: Connection = self.get_connection()?;
+        let where_clause = vec![
+            format!("state = {}", PaymentState::RefundPending as i8),
+            "refund_tx_id IS NULL".to_string(),
```

Comment:
For incoming chain swaps we still need to return refundable as long as the refund_tx is not confirmed.
I don't know if the refund_tx_id is there also when not confirmed yet.